doc/releases/changelog-dev.md

```diff
@@ -28,8 +28,10 @@
   [(#4191)](https://github.com/PennyLaneAI/pennylane/pull/4191)
 
 * Added a conversion function `jordan_wigner` that converts a fermionic operator (`FermiWord`) to a qubit 
-  operator (`PauliSentence`) using the Jordan-Wigner mapping.
+  operator (`PauliSentence`) using the Jordan-Wigner mapping. It also includes the behaviour of the existing
```

Comment:
Is the output actually a `PauliSentence` object?

Reply:
Ah, good point. Either PauliSentence or Operator, default is Operator.